CPP-Examples.cpp

```diff
@@ -11,16 +11,17 @@ int main()
 
```

Comment:
[Banned C function detected](https://github.com/Microsoft/DevSkim/blob/master/guidance/DS154189.md#DS154189) 7 times
These functions are historically error-prone and have been associated with a significant number of vulnerabilities. Most of these functions have safer alternatives, such as replacing 'strcpy' with 'strlcpy' or 'strcpy_s'.